src/intTest/resources/inbound_acceptance_test_data/rejection/1.fhir.json

```diff
@@ -0,0 +1,27 @@
+{
+  "resourceType": "Parameters",
+  "parameter": [
+    {
+      "name": "patient",
+      "resource": {
+        "resourceType": "Patient",
+        "generalPractitioner": [
+          {
+            "reference": "Practitioner/4826940,281"
```

Comment:
Where has this `Practitioner/` appeared from? For those more experience with FHIR is it standard? Would it be reference another resource in a bundle or something? This is different from the example and the documentation already provided to the test partner.

The PDS do something different also. We don't use ODS code though and will never know the period

`"generalPractitioner": [
          {
            "id": "254406A3",
            "type": "Organization",
            "reference": "https://directory.spineservices.nhs.uk/STU3/Organization/Y12345",
            "identifier": {
              "system": "https://digital.nhs.uk/services/organisation-data-service",
              "value": "Y12345",
              "period": {
                "start": "2020-01-01",
                "end": "2021-12-31"
              }
            }
          }
        ],`

Reply:
done